webapp/src/lib/middleware/checkonboarded.ts

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+import { dynamicResponse } from 'misc/dynamicresponse';
+
+const ONBOARDING_ALLOWED_PATHS = new Set([
+	'/onboarding',
+	'/onboarding/configuremodels',
+	'/account.json', //Account context refreshed from here
+	'/team/models.json'
+]);
```

Comment:
I made a change here to not redirect certain pages. /account.json is where the accountcontext is refreshed from, and where the csrf token comes from. This was breaking the ability to get a valid csrf token so I couldn't submit the forms. Similar story for /team/models.json.

Reply:
👍🏼 thanks!